frontend.java

```diff
@@ -0,0 +1,540 @@
+		//name
+		//Make sure the whole name is letters.
+		boolean valid = true;
+		for (char ch : name.toCharArray()){
+	        int asciiVal = (int)ch;	
+			//if((65 <= asciiVal <= 90) || (97 <= asciiVal <= 122) || (asciiVal == 32)){
+			if((asciiVal >= 65) || (asciiVal <= 90) || (asciiVal >= 97) || (asciiVal <=122)){
+				// a - z
+				// space
+				// A - Z
+				valid=true;
+			}else{
+				valid=false;
+			}
+	    }
+		return valid;
+	}
+	public static boolean accountNumber(String number){
+		//covert the string to integers
+		boolean valid = true;
+		try{
+			int x =Integer.parseInt(number);
+		}catch(NumberFormatException e){
+			valid = false;
+		}
+		//check length make sure the account is the proper length
+		int lengthOfAccount = number.length();
+		if(lengthOfAccount != 8){
+			if(lengthOfAccount > 8){
+				valid = false;
+			}
+		}
+		return valid;
+	}
+	public static boolean accountExists(String number){
+		//Check Account File for the account.
+		
+		//Check length, if its less or equal to 8. 
+		//if less add zeros.
+		if(number.length() == 8){
+			BufferedReader br = null;
+			try {
+	 
+				String sCurrentLine;
+	 
+				br = new BufferedReader(new FileReader("./src/valid_accounts.txt"));
+	 
+				while ((sCurrentLine = br.readLine()) != null) {
+					if(sCurrentLine == number){
+						return true;
+					}
+				}
+	 
+			} catch (IOException e) {
+				e.printStackTrace();
+			}
+		}else if(number.length() > 8){
+			return false;
+		}else{
+			String newAccountNumber = addZeros(number, 8);
+			//check the newAccountNumber
+			BufferedReader br = null;
+			try {
+	 
+				String sCurrentLine;
+	 
+				br = new BufferedReader(new FileReader("./src/valid_accounts.txt"));
+	 
+				while ((sCurrentLine = br.readLine()) != null) {
+					if(sCurrentLine == newAccountNumber){
+						return true;
+					}
+				}
+	 
+			} catch (IOException e) {
+				e.printStackTrace();
+			}
+		}
+		return false;
+	}
+	public static boolean accountAmount(String number){
+		try{
+			int x =Integer.parseInt(number);
+		}catch(NumberFormatException e){
+			return false;
+		}
+		return true;
+	}
+	public static String addZeros(String number, int size){
+		//get the number and subtrack it by 8 and add that many zeros
+		int length = number.length();
+		int additionalZeros = size - length;
+		for( int i = 0 ; i < additionalZeros ; i++ ){
+			number = "0" + number;
+		}
+		return number;
+	}
+	public static boolean lowerLimit(String amount){
+		boolean valid = false;
+		try{
+			int amountInt = Integer.parseInt(amount);
+			if(amountInt > 0){
+				valid = true;
+			}
+		}catch(NumberFormatException e){
+			//should not reach here because we validate before.
+			System.out.println("Error 0001");
+			System.exit(1);
+		}
+		return valid;
+	}
+	public static boolean upperLimit(String amount){
+		boolean valid = false;
+		try{
+			int amountInt = Integer.parseInt(amount);
+			if(frontend.type == 1){
+				//retail mode
+				if(amountInt > 100000){//above 1,000.00
```

Comment:
I don't quite understand your comment `//above 1,000.00`  That comment states you expect a double/float, however you have handling integers.  Could you please elaborate?